README.md

```diff
@@ -1,14 +1,66 @@
-# FluentUI Apple
-Fluent UI Apple is a repository containing native UIKit and AppKit controls aligned with Microsoft's Fluent UI design language. 
+# Fluent UI Apple
+Fluent UI Apple is a library containing native UIKit and AppKit controls aligned with [Microsoft's Fluent UI design system](https://www.microsoft.com/design/fluent/#/). 
 
-![CI](https://github.com/microsoft/fluentui-apple/workflows/CI/badge.svg?branch=master)
+![Build Status](https://github.com/microsoft/fluentui-apple/workflows/CI/badge.svg?branch=master)
+![License](https://img.shields.io/github/license/Microsoft/fluentui-apple)
+[![Carthage Compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+[![CocoaPods Compatible](https://img.shields.io/cocoapods/v/MicrosoftFluentUI)](https://cocoapods.org/pods/MicrosoftFluentUI)
+![Platform](https://img.shields.io/cocoapods/p/MicrosoftFluentUI.svg?style=flat)
 
 ## Getting Started
-To get started developing in this repository for iOS, please check out [the iOS readme file](ios/README.md).
+### Install and use FluentUI
 
-To get started developing in this repository for macOS, please check out [the macOS readme file](macos/README.md).
+#### Requirements
 
-# Contributing
+- iOS 11+ or macOS 10.13+
+- Xcode 11+
+- Swift 5.0+
+
+#### Using Carthage
+
+To integrate FluentUI using Carthage, specify it in your Cartfile:
+
+```
+github "Microsoft/fluentui-apple" ~> X.X.X
+```
+
+then follow the Carthage [integration steps](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application) to add the `FluentUI.framework` into your Xcode project
+
+#### Using CocoaPods
+
+To get set up with CocoaPods visit their [getting started guide](https://guides.cocoapods.org/using/getting-started.html).
+
+To integrate FluentUI into your Xcode project using CocoaPods, specify it in your Podfile:
+```ruby
+pod 'MicrosoftFluentUI', '~> X.X.X'
+```
+
+#### Manual installation
+
+- Download the latest changes from the [FluentUI for Apple](https://github.com/microsoft/fluentui-apple) repository.
+- Move the `fluentui-apple` folder into your project folder.
+- Move the relevant `FluentUI.xcodeproj` into your Xcode project depending on which platform you want to support.
+- In Xcode select your project -> your target -> General -> Embedded Binaries -> add `FluentUI.framework`.
+
+#### Swift Package Manager
+As of this writing, the version of Swift Package Manager shipped with the latest Xcode does not support packages that require resource bundles. As Fluent UI Apple does  require resource bundles, we do not currently support Swift Package Manager.
+
+### Import and use the library
```

Comment:
Is the word "library" correct here? At the least it's confusing to me. But then my related question is that these instructions are specific to using it as a framework and not a static library?

Reply:
Reworded.